### PR TITLE
[WIP] +pokt.network

### PR DESCRIPTION
**Pull request by Olshansk**
Adding the `pocket` binary to interface with full Pocket nodes that are part of pokt.network.

Evidence of passing `pkt test`:

Screenshot 2023-03-25 at 11 51 40 AM

### CODE REVIEW

File: projects/pokt.network/package.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+distributable:
+  url: https://github.com/pokt-network/pocket-core/archive/refs/tags/RC-{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: https://github.com/pokt-network/pocket-core/tags
```
Comment (mfts) [inline projects/pokt.network/package.yml]: Only the org/repo without GitHub.com and https

**Comment by Olshansk** [comment]
Both of the issues were resolved by:

1. Making sure my for has a `main` branch
2. Replacing `github: https://github.com/pokt-network/pocket-core/tags
[Member](https://github.com/teaxyz/pantry/commits?author=mfts)` with `github: pokt-network/pocket-core/tags
[Member](https://github.com/teaxyz/pantry/commits?author=mfts)`

`pkg build` and `pkg test` succeeded
---

Before I committed my changes, I was hitting the following issues with my uncommited branch:

Screenshot 2023-03-25 at 11 29 59 AM

After committing my changes, to create this draft PR, I am hitting these changes:

Screenshot 2023-03-25 at 11 30 04 AM

---

File: projects/pokt.network/package.yml (hunk 1 above)
Comment (Olshansk) [reply projects/pokt.network/package.yml]: Works!

**Comment by Olshansk** [comment]
Thank you @mfts and @mxcl! Excited to keep telling all my friends about tea :)

**Pull request merged**